neurom/ezy/neuron.py

```diff
@@ -111,6 +111,7 @@ def __init__(self, filename, iterable_type=np.array):
         self._nrn = load_neuron(filename, set_tree_type)
         self.soma = self._nrn.soma
         self.neurites = self._nrn.neurites
+        self.filename = self._nrn.id
```

Comment:
Yes, I think it is better if we pass the path strip to the initial stage. I will update the core.neuron.Neuron instead.